index.html

```diff
@@ -0,0 +1,416 @@
+                    more days you need to work to 
+                    reach your financial goal.
+                </div>
+                <div class="card__persona">
+                    <img src="./assets/avatar.png" alt="" class="card__avatar">
+                    <div class="card__professional-details">
+                        <p class="card__name">Regina Miles</p>
+                        <p class="card__occupation">Design</p>
+                    </div>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card__ratings">
+                    <img src="./assets/star-full.svg" alt="">
+                    <img src="./assets/star-full.svg" alt="">
+                    <img src="./assets/star-full.svg" alt="">
+                    <img src="./assets/star-full.svg" alt="">
+                    <img src="./assets/star-border.png" alt="">
+
+                </div>
+                <div class="card__message">
+                    Slate helps you see how many 
+                    more days you need to work to 
+                    reach your financial goal.
+                </div>
+                <div class="card__persona">
+                    <img src="./assets/avatar-2.png" alt="" class="card__avatar">
+                    <div class="card__professional-details">
+                        <p class="card__name">Regina Miles</p>
+                        <p class="card__occupation">Design</p>
+                    </div>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card__ratings">
+                    <img src="./assets/star-full.svg" alt="">
+                    <img src="./assets/star-full.svg" alt="">
+                    <img src="./assets/star-full.svg" alt="">
+                    <img src="./assets/star-full.svg" alt="">
+                    <img src="./assets/star-border.png" alt="">
+
+                </div>
+                <div class="card__message">
+                    Slate helps you see how many 
+                    more days you need to work to 
+                    reach your financial goal.
+                </div>
+                <div class="card__persona">
+                    <img src="./assets/avatar-3.png" alt="" class="card__avatar">
+                    <div class="card__professional-details">
+                        <p class="card__name">Regina Miles</p>
+                        <p class="card__occupation">Design</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="team">
+        <div class="note">
+            <h1 class="note__title note__title--text-big">
+                Meet Our <br>
+                Team
+            </h1>
+            <p class="note__description note__description--small">
+                Problems trying to resolve the 
+                conflict between <br class="desktop--line-break"> the two major realms of 
```

Comment:
I thought if we are using modifier here, it should be like `class="desktop desktop--line-break"` not using a modifier only.